### PR TITLE
Introduce publishing workflows configuration and modules

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Adds numerous contrib modules and config to support the publishing workflows. This also includes content type modeling for person, location, press release, notifications, basic pages, and some paragraphs work.

Included in this is a full export of the config from the local development site.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | yes
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Medium
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

File: ecms_base/config/install/openid_connect.settings.generic.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-enabled: true
+enabled: false
```
Comment (pfrilling) [inline ecms_base/config/install/openid_connect.settings.generic.yml:1]: These OIDC configuration needs to be reverted.

File: ecms_base/config/install/openid_connect.settings.facebook.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+enabled: false
```
Comment (pfrilling) [inline ecms_base/config/install/openid_connect.settings.facebook.yml:1]: The OIDC configuration doesn't need to be installed here.

File: ecms_base/config/install/search.page.node_search.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+langcode: en
```
Comment (pfrilling) [inline ecms_base/config/install/search.page.node_search.yml:1]: I don't think we need to include the core search configuration here.

File: ecms_base/config/install/user.settings.yml [hunk 4]
```diff
@@ -9,7 +9,7 @@ notify:
   register_admin_created: true
   register_no_approval_required: true
   register_pending_approval: true
-register: admin_only
```
Comment (pfrilling) [inline ecms_base/config/install/user.settings.yml:12]: This needs to remain admin_only

**Review by pfrilling — CHANGES_REQUESTED** [review]
There is a lot of configuration here that should be either managed by features or doesn't need to be included. 

I think we should piece meal out each content type and make a PR for a feature module with the necessary configuration included for that content type. Then, the profile just needs to install the feature module and all of the configuration will install.

**Comment by pfrilling** [comment]
@ben-hamelin, I created a new PR that I _believe_ is a better approach at getting the content types setup as features. See #7.

**Comment by ben-hamelin** [comment]
Closing this in favor of features PRs.

**Pull request closed without merging**